rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java

```diff
@@ -1280,6 +1449,95 @@ public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_after_RS
         Assert.assertEquals(co.rsk.core.Coin.fromBitcoin(valueToSend).asBigInteger(), result);
     }
 
+    @Test
+    public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_surpasses_locking_cap_should_refund_user_before_RSKIP293() throws BlockStoreException, BridgeIllegalArgumentException, IOException {
```

Comment:
I think we are only missing one test pre RSKIP that sends funds to both active and retiring federation, locking cap is exceeded, and only funds sent to the active federation are refunded.